src/lib.rs

```diff
@@ -262,7 +262,7 @@ pub(crate) fn get_artifact_config(package: Package, artifact: Artifact) -> CTRCo
     // For now, assume a single "kind" per artifact. It seems to be the case
     // when a single executable is built anyway but maybe not in all cases.
     let name = match artifact.target.kind[0].as_ref() {
-        "bin" | "lib" | "rlib" | "dylib" if artifact.target.test => {
+        "bin" | "lib" | "rlib" | "dylib" if artifact.profile.test => {
```

Comment:
Looks okay to me; the `cargo_metadata` docs are a little unclear (to me) about the difference but I guess `Whether or not this target is tested by default by cargo test` would in fact be true for most `lib` targets so I'm surprised I never noticed this before...

Reply:
Seems to also be true for binaries (or, at least, my binary), so I do wonder how it was never caught before!